apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -805,34 +1084,48 @@ int wifi_manager_itc(int argc, FAR char *argv[])
 	itc_wifi_manager_set_mode_p();
 
 	itc_wifi_manager_get_mode_p(); // set softap mode inside this function
-
 	itc_wifi_manager_connect_disconnect_ap_p(); // change to station mode and try to connect to ap
 
 	itc_wifi_manager_scan_ap_p(); // Reinitialized wifi manager with the callback hander for scan results
 
+	WIFITEST_WAIT;
```

Comment:
Instead of calling WIFITEST_WAIT here, it would be better to calling it inside itc_wifi_manager_scan_ap_p().
And I think wifi manager should be deinitialized when each function(itc_wifi*) is ended.